sklearn/mixture/tests/test_gmm.py

```diff
@@ -216,46 +217,51 @@ def test_sample(self, n=100):
         g.covars_ = np.maximum(self.covars[self.covariance_type], 0.1)
         g.weights_ = self.weights
 
-        samples = g.sample(n)
```

Comment:
I guess this is fine, but it might have been easier / shorter to use a decorator on the whole function. But don't worry about it now.